_about/computation.md

```diff
@@ -2,27 +2,26 @@
+3. Software dependencies can be can be staged with the job, distributed via containers, or installed on the read-only distributed OASIS filesystem (which an also support software modules). Statically-linked binaries are ideal. However, dynamically linked binaries with standard library dependencies, built for 64-bit Red Hat Enterprise Linux (RHEL) version 6 or 7 will also work. OSG can support some licensed software (like Matlab, Matlab-Simulink, etc.) where compilation allows execution without a license, or where licenses still accommodate multiple jobs and are not node-locked.
+4. Input and output data for each job should be <20 GB to allow them to be pulled in by the jobs, processed and pushed back to the submit node. Note that the OSG Virtual Cluster does not currently have a global shared file system, so jobs with such dependencies will not work. Projects with many TBs of data can be distributed with significant scalability, beyond the capacity of a single cluster, if subsets of the data are accessed across numerous jobs.
 
 The following are examples of computations that are a great match for OSG:
 
 1. parameters sweeps, parameter optimizations, statistical model optimizations, etc. (as pertains to many machine learning approaches)
 2. molecular docking and other simulations with numerous starting systems and/or configurations
-3. image processing (including medical images with non-restricted data), satellite images, etc.)
+3. image processing (including medical images with non-restricted data), satellite images, etc.
 4. many genomics/bioinformatics tasks where numerous reads, samples, genes, etc., might be analyzed independent of one another before bringing results together
 5. text analysis
 And many others!
 
 The following are examples of computations that are not good matches for OSG:
 
-1. Tightly coupled computations, for example MPI-based communication, will not work well on OSG due to the distributed nature of the infrastructure.
-2. Computations requiring a shared file system will not work, as there is no shared filesystem between the different clusters on OSG.
-3. Computations requiring complex software deployments are not a good fit. There is limited support for distributing software to the compute clusters, but for complex software (though containers may be helpful!), or licensed software, deployment can be a major task.
-
+1. Tightly coupled computations, for example MPI-based multi-node communication, will not work well on OSG due to the distributed nature of the infrastructure.
+2. Computations requiring a shared filesystem will not work, as there is no shared filesystem between the different clusters on OSG.
+3. Computations requiring complex software deployments or restrictive licensing are not a good fit. There is limited support for distributing software to the compute clusters, but for complex software (though containers may be helpful!), or licensed software, deployment can be a major task.
```

Comment:
```suggestion
3. Computations requiring complex software deployments or restrictive licensing are not a good fit. There is limited support for distributing software to the compute clusters (containers can be helpful!), but for complex software or licensed software, deployment can be a major task.
```
Might read less weird, although it still could be better.

Perhaps licensed software should be a separate bullet? For complex software, deployment is just a major task; for some licensed software, deployment is legally impossible.